.gitignore

```diff
@@ -24,4 +24,5 @@ npm-debug.log
 # Alternatively, you may comment the line below and commit the
 # secrets files as long as you replace their contents by environment
 # variables.
-/config/*.secret.exs
\ No newline at end of file
+/config/*.secret.exs
+.elixir_ls/
```

Comment:
With `echo '\n.elixir_ls/' >> .gitignore` the`\n` didn't quite work, like it sometimes does, and it just hard coded it, so I just fixed it via Github.